{"uuid":"bcfb9f3e-e352-4e81-a5ee-a425bba40202","historyId":"a890d6742cb3394a1c5f59ebf3d2c24a","fullName":"com.automationtestingrestassured.CRUD.TC_CreateBooking.testCreateBookingNegative","labels":[{"name":"package","value":"com.automationtestingrestassured.CRUD.TC_CreateBooking"},{"name":"testClass","value":"com.automationtestingrestassured.CRUD.TC_CreateBooking"},{"name":"testMethod","value":"testCreateBookingNegative"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"APIAutomationFramework"},{"name":"subSuite","value":"com.automationtestingrestassured.CRUD.TC_CreateBooking"},{"name":"host","value":"DESKTOP-PRQP6E5"},{"name":"thread","value":"2944@DESKTOP-PRQP6E5.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"owner","value":"TM"},{"name":"severity","value":"critical"}],"links":[],"name":"testCreateBookingNegative","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.ExceptionInInitializerError","trace":"java.lang.ExceptionInInitializerError\r\n\tat com.automationtestingrestassured.base.BaseTest.setConfig(BaseTest.java:35)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: java.lang.RuntimeException: com.codoid.products.exception.FilloException: Workbook is not found - /src/main/java/com/automationtestingrestassured/resource/FillowExcel.xlsx\r\n\tat com.automationtestingrestassured.utils.FillowUtil.fetchDatafromExcel(FillowUtil.java:25)\r\n\tat com.automationtestingrestassured.endPoints.APIConstants.<clinit>(APIConstants.java:25)\r\n\t... 24 more\r\nCaused by: com.codoid.products.exception.FilloException: Workbook is not found - /src/main/java/com/automationtestingrestassured/resource/FillowExcel.xlsx\r\n\tat com.codoid.products.fillo.Fillo.getConnection(Fillo.java:46)\r\n\tat com.automationtestingrestassured.utils.FillowUtil.fetchDatafromExcel(FillowUtil.java:17)\r\n\t... 25 more\r\n"},"stage":"finished","description":"Verify booking is created with no payload","steps":[],"attachments":[],"parameters":[],"start":1720248156182,"stop":1720248156198}